examples/nirfsa/getting-started-iq.py

```diff
@@ -64,7 +64,7 @@ def check_for_warning(response, vi):
     """Print to console if the status indicates a warning."""
     if response.status > 0:
         warning_message = client.ErrorMessage(
-            nirfsa_types.ErrorMessageRequest(vi=vi, error_code=response.status)
+            nirfsa_types.ErrorMessageRequest(vi=vi, status_code=response.status)
```

Comment:
For nirfsa, the `ErrorMessageRequest`'s field name is `status_code` and not `error_code`.